source/overview/mattermost-cloud-overview.rst

```diff
@@ -37,21 +37,21 @@ Frequently Asked Questions (FAQs)
 How much does Mattermost Cloud cost?
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-Mattermost Cloud Professional is free for up to 10 users. After 10 users, it is $10 USD per user, per month as a pay-as-you-go subscription. If you have over 1000 users please contact us.
+Mattermost Cloud Professional is $10 USD per user, per month as a pay-as-you-go subscription. If you have over 1000 users please contact us.
```

Comment:
@wiersgallak - Does this line need to change as well?

Reply:
I chose to keep this to what will be released as the new process.